code/modules/organs/subtypes/xenos.dm

```diff
@@ -61,6 +82,12 @@
 	icon_state = "acidgland"
 	organ_tag = O_ACID
 
+	organ_verbs = list(
```

Comment:
You sure these all belong on the same organ? Last I checked the different xenos aren't all supposed to have all three of those powers.

Reply:
This only gives the Drone the two spits, which considering how easily it dies, and the fact the AI drones spit, isn't the biggest of concerns in my opinion. If it needs to be changed later, I can make a new drone organ.